install.sh

```diff
@@ -1,6 +1,14 @@
-#!/bin/bash
+#!/bin/bash -e
 
 dir=~/.local/share/gnome-shell/extensions/vpn-indicator@howdoicomputer.fastmail.com
 
-rm -rf $dir
+if [ -d $dir ]; then
```

Comment:
i kept your remove, but put in a directory check - this ensures that bash does not fail out before attempting to remove a dir